src/trackerHook.js

```diff
@@ -18,7 +18,7 @@ export const usePromiseTracker = (outerConfig = defaultConfig) => {
       setInternalPromiseInProgress(true);
       setPromiseInProgress(true);
     }
-  }, config)
+  }, [config])

```

Comment:
I'm also getting the same error if i pass parameter like config. but if i pass it like [config], continuous API call getting triggering.
I have API call in useEffect.